includes/utils.php

```diff
@@ -1139,6 +1139,7 @@ function post_args_allow_list( $post_args ) {
 		'post_title',
 		'post_excerpt',
 		'post_status',
+		'post_type',
```

Comment:
Also noticed we don't include the `comment_status` or `ping_status` arguments here, though those are default ones `wp_insert_post` accepts. Were those left out intentionally?

Reply:
I've added them in 835b1ca38723f1ce4d323b2529c0743857091a4d.

I suspect I lost my spot going through the arguments and they were skipped due to an oversight, along with the post type.